modelview/templates/modelview/base.html

```diff
@@ -22,7 +22,7 @@
 want to compare. OEP will give us tables (views) that facilitate the comparison.
 <hr>
 {% endblock %}
```

Comment:
```suggestion
{% endblock main-right-sidebar-content %}
```

Reply:
>`- {% endblock %}`
>`+ {% endblock main-right-sidebar-content %}`

@Bachibouzouk. Right now, only endblocks in `base/templates/base/base.html` have a more verbose naming, because, I would have to update all endblocks by hand with the more verbose naming. I can do that, if you agree with the more verbose naming. Personally if the code blocks are larger this helps my a lot, but can be an overkill if the code blocks are small. On the other hand IMO it makes sense to have everywhere the same... What do you think? Do you like this more verbose naming? Shall we implement it all templates?